addon/templates/components/notification.hbs

```diff
@@ -0,0 +1,19 @@
+{{#fbf-item borderTop=true}}
+  <div class="fbf-item__primary-icon fbf-item--center-content-horizontally">
+    {{activities/icon-view model=model type=model.type}}
+  </div>
+  <div class="fbf-item__content">
+    {{#fbf-text lines=1}}
+      A Goal was updated!
+    {{/fbf-text}}
+    {{#fbf-meta-text}}
+      {{#fbf-text light=true lines=1}}
+        {{model.title}}
+      {{/fbf-text}}
+      {{fbf-text-divider}}
+      {{#fbf-text light=true}}
+        Just now
```

Comment:
The goal model should have an `updated_at` property (at least in the backend). We don't have to use it yet, but it would be nice to see when it was updated.